frontend/src/pages/ReviewWritingPage/progressBar/components/MobileProgressBar/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useRef } from 'react';
+
+import NavigateNextIcon from '@/assets/navigateNext.svg';
+import useStepList from '@/pages/ReviewWritingPage/progressBar/hooks/useStepList';
+import { Direction } from '@/pages/ReviewWritingPage/types';
+
+import * as S from './styles';
+
+interface MobileProgressBarProps {
+  currentCardIndex: number;
+  handleCurrentCardIndex: (direction: Direction) => void;
+}
+
+const MobileProgressBar = ({ currentCardIndex, handleCurrentCardIndex }: MobileProgressBarProps) => {
+  const { stepList } = useStepList({ currentCardIndex });
+
+  const stepRefs = useRef<HTMLDivElement[]>([]);
+
+  useEffect(() => {
```

Comment:
```suggestion
  useEffect(() => {
    if (stepRefs.current[currentCardIndex]) {
      const element = stepRefs.current[currentCardIndex];

      setTimeout(() => {
        element.scrollIntoView({
          behavior: 'smooth',
          inline: 'center',
        });
      }, 0);
    }
  }, [currentCardIndex, stepRefs]);
```

setTimeout을 사용하니, 다음 버튼 클릭 시 가운데로 오네요!! 😮
리뷰 작성 폼의 애니메이션과 프로그레스 바 애니메이션이 충돌 되어서 그런건지, 
정확한 이유는 모르겠네요 🥲

Reply:
GPT의 설명 첨부할게요

>카드가 처음 열릴 때 해당 요소가 아직 DOM에 완전히 렌더링되지 않았을 수 있습니다. 특히, React에서 상태가 변경되면서 비동기적으로 DOM이 업데이트되는데, scrollIntoView가 호출되는 시점에 해당 요소가 DOM에 아직 존재하지 않으면 작동하지 않을 수 있습니다.